systers_portal/meetup/tests/test_views.py

```diff
@@ -128,7 +126,39 @@ def test_post_add_meetup_view(self):
                 'description': "It's a test meetup."}
         response = self.client.post(url, data=data)
         self.assertEqual(response.status_code, 302)
-        new_meetup = Meetup.objects.get()
+        new_meetup = Meetup.objects.get(slug='bartest')
         self.assertTrue(new_meetup.title, 'BarTest')
         self.assertTrue(new_meetup.created_by, self.systers_user)
         self.assertTrue(new_meetup.meetup_location, self.meetup_location)
+
+
+class EditMeetupView(MeetupLocationViewBaseTestCase, TestCase):
+    def test_get_edit_meetup_view(self):
+        """Test GET edit meetup"""
+        wrong_url = reverse("edit_meetup", kwargs={'slug': 'foo', 'meetup_slug': 'foo'})
+        response = self.client.get(wrong_url)
+        self.assertEqual(response.status_code, 403)
+
+        url = reverse("edit_meetup", kwargs={'slug': 'foo', 'meetup_slug': 'foo-bar-baz'})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 403)
+
+        self.client.login(username='foo', password='foobar')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'meetup/edit_meetup.html')
+
+    def test_post_edit_meetup_view(self):
+        """Test POST edit meetup"""
+        wrong_url = reverse("edit_meetup", kwargs={'slug': 'foo', 'meetup_slug': 'foo'})
+        response = self.client.post(wrong_url)
+        self.assertEqual(response.status_code, 403)
+
+        url = reverse("edit_meetup", kwargs={'slug': 'foo', 'meetup_slug': 'foo-bar-baz'})
+        response = self.client.post(url)
+        self.assertEqual(response.status_code, 403)
+
+        data = {'name': 'Bar', 'slug': 'bar', 'venue': 'edit test address'}
+        self.client.login(username='foo', password='foobar')
+        response = self.client.post(url, data=data)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Here the status code should be `302` so that I can check `url.endswith('/meetup/foo/bar/')` but the status code is `200`.

Do you have any thoughts why status code is 200 instead of 302. Because of it, I am unable to check `url ends with` due to which test coverage is 99%.